Makefile

```diff
@@ -83,6 +83,25 @@ kubeval: _site
 	! grep -C3 -P "invalid|\t\*" filtered.out
 	rm filtered.out
 
+helm-tests: vendor bin/helm helm-tests/fv.test versions-values.yaml
```

Comment:
Ideally I would add this to the `ci` target as well, but the dependency on `RELEASE_STREAM` at the moment prevents that.

Reply:
Dang, that's annoying. Well lets doe whatever `kubeval` does which seems like it only tests `master`.